stdlib/public/SDK/Dispatch/Source.swift

```diff
@@ -299,35 +303,46 @@ public extension DispatchSourceFileSystemObject {
 }
 
 public extension DispatchSourceUserDataAdd {
-	/// @function mergeData
+	/// @function add
```

Comment:
Ah, if you're fixing these up, you should probably use Swift doc comment syntax. Swift does not recognize HeaderDoc.